src/screens/Ticketing/Purchase/Payment/SelectPaymentMethod/index.tsx

```diff
@@ -103,6 +89,16 @@ const SelectPaymentMethod: React.FC<Props> = ({
 
   const {user} = useAuthState();
   const {theme} = useTheme();
+  const {paymentTypes} = useFirestoreConfiguration();
+
+  const defaultPaymentOptions: SavedPaymentOption[] = paymentTypes.map(
```

Comment:
This seems OK, but just to be sure check how it is handled when the `getSelectedPaymentMethod(previousPaymentMethod)` below has a `previousPaymentMethod` which is not in the firestore config. If "vipps" is removed from Firestore it should not be still usable for those who has it stored as `previousPaymentMethod`.

Reply:
Good catch, I was expecting the same but because that saved payment option comes from the server maybe it is filtered. But I will do that as well, thanks.

Reply:
Just note that there is a difference between `recurringPayments` which are saved payment cards which comes from the server, and the `previousPaymentMethod` which does not come from the server, but is stored locally on the phone storage. However for implementation purposes it is not a big difference whether it comes from a server or phone storage.

Reply:
I have covered both cases now, please review it ;)